src/index.css

```
@@ -3,6 +3,274 @@
 @custom-variant dark (&:is(.dark *));
 
+/* ============================================================
+   VataApp — Core tokens
+   A warm, earthy palette layered on shadcn/ui (new-york) tokens.
+   Light is the :root default; add class="dark" on <html> for dark.
+   ============================================================ */
+
+/* Self-hosted font files. */
+
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.7.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)